dotnet-8.0.404.yaml

```diff
@@ -0,0 +1,99 @@
+package:
+  name: dotnet-8.0.404
+  version: 8.0.404
+  epoch: 1
+  description: ".NET SDK"
+  copyright:
+    - license: MIT
+  resources:
+    cpu: 2
+    memory: 32Gi
+  dependencies:
+    runtime:
+      - dotnet-8-sdk-default
+      - icu
+
+environment:
+  environment:
+    DOTNET_CLI_TELEMETRY_OPTOUT: 1
+    DISABLE_CROSSGEN: true
+    DisableArcade: 1
+  contents:
+    packages:
+      - bash-binsh
+      - brotli-dev
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - clang-15
+      - clang-15-default
+      - cmake
+      - curl
+      - dotnet-bootstrap-8
+      - glibc-locale-en
+      - icu-dev
+      - krb5-dev
+      - libunwind
+      - libunwind-dev
+      - llvm15
+      - llvm15-cmake-default
+      - llvm15-dev
+      - llvm15-tools
+      - lttng-ust-dev
+      - ncurses-dev
+      - nodejs-22
+      - openssl-dev
+      - posix-libc-utils
+      - python3
+      - rapidjson-dev
+      - samurai
+      - wolfi-base
```

Comment:
this is just for me, but under what circumstances do we need build-base and wolfi-base?

Reply:
this is copy & paste from our existing dotnet-8.yaml and dotnet-9.yaml packaging. It is likely that many of these are redundant and we can trim them down.